cmd/aro/update_ocp_versions.go

```diff
@@ -30,19 +32,38 @@ func getLatestOCPVersions(ctx context.Context, log *logrus.Entry) ([]api.OpenShi
 	dstRepo := dstAcr + acrDomainSuffix
 	ocpVersions := []api.OpenShiftVersion{}
 
+	// INSTALLER_IMAGE_DIGESTS is the mapping of a minor version to
+	// the aro-installer wrapper digest.  This allows us to utilize
+	// Azure Safe Deployment Practices (SDP) instead of pushing the
+	// version tag and deploying to all regions at once.
+	var installerImageDigests map[string]string
+	jsonData := []byte(os.Getenv("INSTALLER_IMAGE_DIGESTS"))
```

Comment:
nit: `"INSTALLER_IMAGE_DIGESTS"` can be a const and then used on lines 48,52 as well